document-generator-company-report/src/main/java/uk/gov/companieshouse/document/generator/company/report/mapping/model/document/items/pscs/items/Psc.java

```diff
@@ -0,0 +1,119 @@
+package uk.gov.companieshouse.document.generator.company.report.mapping.model.document.items.pscs.items;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.List;
+import uk.gov.companieshouse.document.generator.company.report.mapping.model.document.items.common.Address;
+import uk.gov.companieshouse.document.generator.company.report.mapping.model.document.items.common.DateOfBirth;
+import uk.gov.companieshouse.document.generator.company.report.mapping.model.document.items.common.Identification;
+
+public class Psc {
+
+    @JsonProperty("address")
+    private uk.gov.companieshouse.document.generator.company.report.mapping.model.document.items.common.Address address;
```

Comment:
full path shouldn't be needed? As it is declared above under import so could just be Address address;